deps/ghostscript.json

```diff
@@ -0,0 +1,20 @@
+{
+    "name": "ghostscript",
+    "config-opts": [
+        "--disable-cups",
+        "--disable-gtk",
+        "--enable-dynamic",
+        "--with-drivers=FILES"
+    ],
+    "cleanup": [
+        "/libexec",
+        "/share/man"
+    ],
+    "sources": [
+        {
+            "type": "archive",
+            "url": "https://github.com/ArtifexSoftware/ghostpdl-downloads/releases/download/gs922/ghostscript-9.22.tar.xz",
```

Comment:
This ghostscript version is one year old with tons of CVE allowing for arbitrary code execution. Latest is 9.25.

Reply:
ok

Reply:
For what is ghostscript required; is this dependency really required?

Reply:
I think it is just for building

Reply:
If its just for building it could have cleanup: ["*"] to avoid shipping it in the final app.